submissions/m-ruslan/js-dom/app.js

```diff
@@ -0,0 +1,86 @@
+document.addEventListener("DOMContentLoaded", () => {
+  console.log("DOM is loaded");
+  loadPageData("info");
```

Comment:
I would put it in a constant, something like "DEFAULT_PAGE", to minimize risk of typos or inconsistent changes throughout the code.

Reply:
Thank you
I've added constant for this